src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpTokenService.cs

```diff
@@ -1,7 +1,8 @@
-﻿namespace SFA.DAS.AdminService.Web.Infrastructure
+﻿using System.Threading.Tasks;
+namespace SFA.DAS.AdminService.Web.Infrastructure
 {
     public interface IRoatpTokenService
     {
-        string GetToken();
+        Task<string> GetToken();
```

Comment:
Instead of doing this could we duplicate the <string> GetToken to the IRoatpApplyTokenService and drop the interface inheritance from that (need to check the StructureMap setup) in that way you can update just the IRoatpTokenService which is the code that our team maintains

Reply:
This has been reverted as discussed